Documentation/getting-started.md

```diff
@@ -0,0 +1,143 @@
+# Getting Started with acbuild
+
+acbuild aims to support a workflow very similar to `Dockerfile`s, but with more
+flexibility and adherence to the Unix tools philosophy, and native image output
+in the modern ACI format. Generally, the workflow consists of a shell script
+(analogous to the `Dockerfile`) driving acbuild to construct an ACI, either
+from scratch or with another image as a base. This ACI is then stored in an
+_image registry_, and fetched by users to execute the applications stored
+within.
+
+The following guide will walk you through building and running a simple ACI
+with acbuild. The ACI will contain nginx, and will serve static files over HTTP.
+
+Note that some of the commands shown here must be run as root.
```

Comment:
The documentation could also start the commands with `$` or `#` to indicate if `root` is necessary.